src/pages/index.js

```diff
@@ -78,11 +162,33 @@ function applyTransition() {
     modal.classList.add("modal__transition");
   });
 }
-
+profileImage.setEventListeners();
 profileModal.setEventListeners();
 imageModal.setEventListeners();
 pictureModal.setEventListeners();
+deleteModal.setEventListeners();
+profilePicModal.setEventListeners();
 
-displayCards();
 enableValidation();
 applyTransition();
+api
+  .getUser()
+  .then((res) => {
+    user.setUserInfo(res);
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+api.getInitCards().then((res) => {
```

Comment:
You can receive all the initial data together using `Promise.all`, so you will save user's id and provide it to Card with constructor arguments and will be able to define if user is card's owner

```js
let userId;

Promise.all([api.getUserInfo(), api.getInitialCards()]).then(([user, cards]) => {
  userInfo.setProfileInfo(userData.name, userData.about);
  userId = user._id;  // set user id
  // initialize cards section
}).catch(() => console.log(...))
```